spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -32,17 +32,33 @@ A repository abstraction aims to significantly reduce the boilerplate code requi
 
 The parent interface in Jakarta Data repository abstraction is DataRepository.
 
-By default, Jakarta Data has support for three interfaces. However, the core is extensible. Therefore, a provider might extend one or more interfaces to a specific data target.
-
-image::02-repositories.png[alt=Repositories types]
```

Comment:
Should this diagram be removed if it is no longer valid?

Reply:
Good catch, I added a commit to remove it